Prototyper/Classes/PrototyperController.swift

```diff
@@ -144,17 +147,19 @@ open class PrototyperController: NSObject {
     }
     
     // MARK: Share App
-    
     private static func shareApp() {
-        guard let instantiatedViewController = UIStoryboard(name: "Share",
-                                                            bundle: Bundle(for: ShareViewController.self)).instantiateInitialViewController() else {
-            return
-        }
+        let instantiatedViewController = UIHostingController(rootView: ShareView().environmentObject(Model()))
+        instantiatedViewController.isModalInPresentation = true
+        self.isFeedbackButtonHidden = true
         topViewController?.present(instantiatedViewController,
                                    animated: true,
                                    completion: nil)
     }
     
+    static func dismissView() {
```

Comment:
If we have a shared model that is passed down as an environment object I would suggest to move that functionality there so every view in the view Hierarchy can dismiss the topViewController without a static call to the `PrototypeController`.